Различные типы атаки у хуманов.

LudwigVonChesterfield opened the pull request:
Как-то странно что рот есть, а кусать им не могу.

![image](https://user-images.githubusercontent.com/17705613/46066372-2c1d7180-c17d-11e8-9bba-52148228b800.png)

кулак:
+1 дамаг от того что было(2-6)
50% того что противник будет повален при крите(6 дамага)
промах только если дамаг выпадет на 1.

ноги:
дамаг(1-5)
75% повалить если крит
промах если выпадает 0, и ещё есть 15% просто так промахнуться дополнительно.
если бьёшь в что-то, что не ГРОИН, НОГИ - оно в большистве случаев направит на них.
если пинаешь лежачего - ему накидывается голоурон.
если пинаешь скованного - повышен шанс опрокидывания.

укус:
дамаг(1-5)
0% повалить
промах если выпадает 0, и 30% просто так промахнуться.
если кусаешь что-то, что не ГОЛОВА, УРКИ - оно в большистве случаев направит на них.
"острые", накидывают кровотечение.
кусать связанными могут только унатхи.

удар током(прерогатива СПУ):
дамаг(1-5) (BURN)
нету промахов.
наносит удары током в соответствии с дамагом(скорее оглушает, чем дамажит).
съедает 30 единиц условного заряда у СПУ за каждую единицу дамага(вот и считайте).

гипноз(Шедоулинги):
дамаг(1-5)(единиц брейндамага)
заставляет ударенного галлюцинировать.
не промахивается.
ударенный слышит страшные звуки.

:cl: Luduk
- rscadd[link]: У хуманов, и ксеноты появились "атаки", отличные от обычного удара рукой. У большинства ксеносов есть укус, пинок(удар ногой), и удар. У унатхов и таяран к этому добавляется "царапанье", дионы, СПУ, шедоулинги, абдукторы, големы не могут кусать.
- rscadd: Унатхи способны своей "слюной", при укусах заражать раны на укушенной конечности.

LudwigVonChesterfield committed:
У всех появилась возможность бить по-разному.

Moggilazz commented:
Сломаны атаки предметами, вместо них идёт выбранная рукопашная атака

LudwigVonChesterfield commented:
:pekaface: Деда, если ты КУСАЕШЬ, ты КУСАЕШЬ, а не бьёшь. Это фича. А с кулаками - рукопашная.

Moggilazz commented:
Пипец, каждый раз на кулак переключаться, ну и вата. А то что эти атаки на мартышек и живность не срабатывают, это тоже задумано так?

LudwigVonChesterfield commented:
А это уже проблема рефакторинга, пока они работают на людей и обезьян. ВНЕ SCOPE OF PR.

LudwigVonChesterfield commented:
Но если замёржат это - могу начать работу и над общими методами получения урона для всех мобов.

Moggilazz commented:
Так на обезьян не работают же

LudwigVonChesterfield commented:
Имел ввиду на Яна.

LudwigVonChesterfield committed:
Обощение проков для зверюшек.

ghost commented:
Сделайте что бы в наручниках можно было ебашить ногами.

LudwigVonChesterfield commented:
Уже сделано.

LudwigVonChesterfield committed:
"Баланс" ченджес в механике боёвки.

- Связанных проще опрокинуть.
- Связанные в руках не могут кусаться(?).
- 50% что связанный подскользнётася при попытке ударить.

Kernstall commented:
Кровотечение за укус накидывается? 
Если прокает крит, то над головой надо проигрывать анимацию вращающегося кубика :pepe:
![image](https://user-images.githubusercontent.com/16035287/46331672-be13f700-c621-11e8-833a-ea746dcc8742.png)

DieSiege commented:
Убери возможность пинаться в наручниках/занерфи это сильно и я замерджу. Иначе это спровоцирует офицеров устраивать еще больше POLICE BRUTALITY на станции.

Flankir commented:
Возможность пинаться в наручниках должна быть но она не должна работать как суперприем.
И кусаться в наручниках не должно быть невозможно если кляп не засунули.

LudwigVonChesterfield commented:
>Убери возможность пинаться в наручниках/занерфи

А в чём тогда разница между типами атаки?

DieSiege commented:
Тогда добавь шанс опрокинуться при попытке бить ногой связанным 🤷‍♂️. Равновесия же никакого когда тебя ведут со связанными руками. Довольно высокий шанс, чтоб он не бил по балансу игровому и не заставлял СБшников играть в Police Brutality, оставляя все же мизерный шанс так сбежать.

LudwigVonChesterfield commented:
Прикрутил 50% на подскальзывания при пинке связанным.

DieSiege commented:
K. Жду коммита.

LudwigVonChesterfield commented:
forcepush my man, forcepush. Можешь читать описание последнего.

DieSiege commented:
Huh. Zametil.
